### ISSUE TITLE
Allocate the specified parameters to optimize

### ISSUE THREAD

**Issue by icharlest** [PaddlePaddle/Paddle#5236]
I face the problem that I didn't find the API which support to optimize the specified parameters.
For example, in the trainer.SGD class, if the parameters are the part of the parameters instead of the whole parameter dictionary, will we just optimize the specified parameters to reduce the loss?

Something like var list in tensorflow optimizer.minimize(, var_list=), just optimize the parameters in the list.

**Comment by ranqiu92** [comment]
You can set is_static of a layer's param_attr True to keep the parameters of this layer unchanged during training.

**Comment by icharlest** [comment]
I don't want to freeze this layer all the time, I just want to freeze this layer sometimes. So, how to do it or can I do it in paddle?

**Comment by ranqiu92** [comment]
Paddle cannot support this in the current version.

**Comment by icharlest** [comment]
I see. Thx
